.travis.yml

```diff
@@ -33,10 +32,18 @@ jobs:
         - yarn lint
         - yarn test
     - stage: deploy
-      name: 'Release recomponents to NPM'
+      name: 'Release recomponents to NPM and deploy storybook to github pages'
       before_script:
         - cd packages/recomponents
       if: branch = master
       script:
         - yarn build
-        - yarn semantic-release
\ No newline at end of file
+        - yarn semantic-release
+        - yarn build-storybook
+        - cd .out
+        - git init
+        - git config user.name "junyong"
```

Comment:
We can't replace this with more generic credentials, can we?

Reply:
I think Can, need a generic GitHub account maybe. 
BTW: I'm using my Github account to `git push` to `gh-pages` branch now.

Reply:
Oh for the `user.name`, I think whatever it is, it should be OK